src/WebUI/Components/ServicesComponent.tsx

```diff
@@ -24,21 +24,32 @@ const ageKey: string = "age-col";
 
 export interface IServicesComponentProperties extends IVssComponentProperties {
     servicesList: V1ServiceList;
+    typeSelections: string[];
+    nameFilter?: string,
     onItemInvoked?: (item?: any, index?: number, ev?: Event) => void;
 }
 
 export class ServicesComponent extends BaseComponent<IServicesComponentProperties, {}> {

```

Comment:
Currently no data is shown when no filter criteria match. IMO, zero data cannot be used here since there is some data but current filter preferences do not match. In case of zero data, there is no data to begin with. Showing zero data here would confuse UX, since zero data prompts to create workloads/services